.eslintrc

```diff
@@ -102,6 +101,30 @@
       "env": {
         "jest": true
       }
+    },
+    {
+      "files": ["src/**", "pages/**", "test/**", "scripts/**"],
+      "rules": {
+        "simple-import-sort/imports": [
+          "warn",
+          {
+            "groups": [
+              // External packages come first.
+              ["^react", "^\\w", "^(?!@cloudscape)@?\\w"],
```

Comment:
let's unify that with the other [PR](https://github.com/cloudscape-design/chat-components/pull/14)